### PR TITLE
fix: add Jatin Kumar

### PR DESCRIPTION
**Pull request by Jatinkumar30**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/813"></a>

### CODE REVIEW

File: public/data/Jatinkumar30.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Jatin Kumar",
+    "type": "personal",
+    "bio": "Open Source Enthusiast and Front end developer!",
+    "avatar": "https://github.com/jatinkumar30.png",
```
Comment (Panquesito7) [inline public/data/Jatinkumar30.json:5]:
```suggestion
    "avatar": "https://github.com/Jatinkumar30.png",
```

File: public/data/Jatinkumar30.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/jatinkumar30",
```
Comment (Panquesito7) [inline public/data/Jatinkumar30.json:9]:
```suggestion
        "url": "https://github.com/Jatinkumar30",
```

Comment (Jatinkumar30) [reply public/data/Jatinkumar30.json]: Updated github url @Panquesito7 please do check it out.

File: public/data/Jatinkumar30.json (hunk 1 above)
Comment (Panquesito7) [inline public/data/Jatinkumar30.json:5]:
```suggestion
    "avatar": "https://github.com/Jatinkumar30.png",
```

**Review by schmelto — APPROVED** [review]

File: public/data/Jatinkumar30.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/JatinRawal30",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Get a grip on Open Source",
+        "date": "January 2022",
+        "icon": "dollar",
```
Comment (Pradumnasaraf) [inline public/data/Jatinkumar30.json:22]:
```suggestion
        "icon": "github",
```
**It's your personal preference what icons you want to keep, but the `GitHub` icon sounds more appealing here :)**

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]

File: public/data/Jatinkumar30.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        "color": "grey",
```
Comment (Pradumnasaraf) [inline public/data/Jatinkumar30.json:23]:
```suggestion
        "icon": "github",
        "color": "green",
```
**It's your personal choice, what icon you want to put into, but I will highly recommend you to use the `GitHub` icon as it sounds more appealing. :)**

File: public/data/Jatinkumar30.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/JatinRawal30",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Get a grip on Open Source",
```
Comment (krishguptadev) [inline public/data/Jatinkumar30.json:20]: // Tense should be matched to description IMHO 

```suggestion
        "title": "Getting a grip on Open Source",
```

File: public/data/Jatinkumar30.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Getting grip on Open Source",
+        "date": "January 2022",
+        "icon": "github",
+        "color": "green",
+        "description": "Getting into open source"
```
Comment (Panquesito7) [inline public/data/Jatinkumar30.json:24]:
```suggestion
        "description": "Getting into Open Source"
```

**Review by Vyvy-vi — APPROVED** [review]
looks good to me

**Review by krishguptadev — APPROVED** [review]

File: public/data/Jatinkumar30.json [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/JatinRawal30",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Getting grip on Open Source",
```
Comment (Panquesito7) [inline public/data/Jatinkumar30.json:20]: @Vyvy-vi, I saw you marked as resolved my other comment. Should we change this to `open-source` or keep it as-is?

Comment (eddiejaoude) [reply public/data/Jatinkumar30.json:20]: this is a personal preference, let the author decide how they want to write it - these types of feedback should be suggestions and not blockers

I will review and merge this as part of a video I am recording today - so please do not merge, I will do this in a few hours

**Review by Pradumnasaraf — APPROVED** [review]

**Review by thepranaygupta — APPROVED** [review]

**Review by eddiejaoude — COMMENTED** [review]
Looks good 👍

**Pull request merged**